docs/dev/datastream_execution_mode.md

```diff
@@ -1,8 +1,6 @@
 ---
 title: "Execution Mode (Batch/Streaming)"
-nav-id: datastream_execution_mode
-nav-parent_id: streaming
-nav-pos: 1
+nav-id: datastream_execution_mode nav-parent_id: streaming nav-pos: 1
```

Comment:
Are all the changes in the file required? Moreover I think this change to the header may break the page.

Reply:
I don't know how those snuck in... 🙈, reverting